webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,17 +1,24 @@
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
 import { InfoCircleIcon } from '@patternfly/react-icons';
-
+import Loading from 'foremanReact/components/Loading';
 import { translate as __ } from 'foremanReact/common/I18n';
 
 import './AnsibleHostDetail.scss';
+import WrappedAnsibleHostInventory from './components/AnsibleHostInventory';
 
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = ({ response, status }) => {
   // https://projects.theforeman.org/issues/32398
-  const [activeTab] = useState('variables');
+  const [activeTab, setActiveTab] = useState('variables');
+  const handleTabClick = (event, tab) => setActiveTab(tab);
+
+  if (status === 'PENDING') {
+    return <Loading />;
```

Comment:
the host details page uses `SkeletonLoader`, how about using it also in ansible?

Reply:
Quick question does the SkeletonLoader should wrap the <Tabs> tag or every <Tab> tag separately?

Reply:
You can wrap the entire component with one `SkeletonLoader` with multiple skeletons
```js 
<SkeletonLoader skeletonProps={{ count: 10 // 10 skeletons }}>
  <Content />
</SkeletonLoader>1
```